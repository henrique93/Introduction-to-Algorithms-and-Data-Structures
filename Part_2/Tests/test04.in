10
close @Jack #Pumpkin
list #Infinity
send @Charlie #Infinity uIJUgCA6HWtef8lYQL4RWViq8novxu
send @Jack #Infinity GUq76QjciIXwhBNujyI5WW9vczNvTj
list #Infinity
list #ModernTimes
send @Charlie #Pumpkin 7AmDt0u1M9ycYqsNdpU4aVBhbYbHUh
close @Jack #Infinity
send @Charlie #ModernTimes KuZW7NR UM6fRH0FM nRONkoZIM3Sz
send @Charlie #Infinity 5w44CQK2nc83PTGVeruUQDxj8bkWag
send @Jack #Pumpkin 3yasSOQSK50x5bc aMqe3n2sAxefbs
send @Jack #Infinity 35lOTCBJBlNP5PDmPnjHtoroETg6sc
send @BuzzLightYear #Infinity GLYUaI11KgcmPMyLGv1Br9QPAqsjsB
list #ModernTimes
close @Charlie #Infinity
close @Charlie #ModernTimes
close @Jack #ModernTimes
send @BuzzLightYear #Pumpkin dEu2Bdse1igqg5pfN9fbnKoPIOxuz 
send @Jack #Pumpkin OD7jUWOzMgQzhdoWuP7uDri4dJ5hJx
send @BuzzLightYear #Pumpkin yOq0kIwtLztmw2T0cz9H9CDYD3IfW7
send @Charlie #Infinity ycU3TXbHA8osUk35GdY6TXVhjFP3Lf
send @Charlie #ModernTimes ChCDhO9FftNAPzl3oW5FqDJ 8U91eE
send @Charlie #Infinity wpOah3z1JPkdrqhJHDubCaODZ0KX9W
send @BuzzLightYear #ModernTimes T0vBZYuOc m50SJSbtTiO7waZmuSez
close @Charlie #Pumpkin
send @BuzzLightYear #Pumpkin H5 fXwfMytU4S9eXQ7ymVNysPrTfLn
close @Jack #Infinity
send @Jack #Pumpkin QGEt7glVESoDNLqeAHrrI6vgBYPhsg
send @Charlie #Pumpkin 7lipHexFcrjUxYBJ3xQnBPmL6pqX1y
send @Charlie #Pumpkin K06Vjxp MUAWcKYKVsYO65jj5egYiZ
send @BuzzLightYear #Infinity IXpxzAlI9s47us5 KlQ7nPsSBKLbAp
list #Infinity
send @Jack #Pumpkin dmJ0lk7yxmQhQ7P4o0EHf7QqfE6xVH
list #Pumpkin
send @BuzzLightYear #ModernTimes zlJ5QmXuY1wd73rNgSmAM2Q9WOtVtc
send @BuzzLightYear #Infinity Xrvb71uePzMkwtEaFJKKb6QLYagzbQ
send @Jack #Infinity IpPDsj0HHzROMrYO4qyYaO4M08jI8u
send @Jack #Infinity 5A016CjG6yB5e0TrzgFBvd7N4AupIl
close @Jack #Infinity
send @Jack #ModernTimes BFvy73COFk9gQp5JQMdbJKrQkSadnY
close @BuzzLightYear #Pumpkin
send @BuzzLightYear #Infinity 4rsGbDjkk4Bgasdw1riFYiyE3Zaimq
send @Charlie #Infinity fs0WpSWAlt7BvRmNqTRNO5dZBd4WEb
send @Charlie #Pumpkin A4pBtdXJV2TMH7mza7elO9mQrKiLNE
send @Charlie #Pumpkin KcM752TmgcKgWPuwSlH0C7xm2elR r
send @BuzzLightYear #Infinity  DWHK036AcET2upIiOuf4LqXd2BIwk
send @BuzzLightYear #Infinity 3Z QMToSmwALWiJq9Im0xW0jAd1ko9
send @Jack #Pumpkin wWZnGk 1sq4kX0UfBJPS3JnvpmeQ2a
send @Charlie #Infinity Ipo3jEikzVEDW4dSAyf8AlB59Z9Afz
send @Charlie #ModernTimes M9yVg 0o2P2Q3zk4s1Xfm2t1gGOvjS
send @Jack #ModernTimes QTcrVxvDgL XIyZd1raKsHCWR7h3j7
close @Charlie #ModernTimes
send @Jack #ModernTimes uBXVChPzDxV3RmoJlsmBAdzVcu31wS
list #ModernTimes
close @BuzzLightYear #Pumpkin
send @Jack #ModernTimes W6yhLxnF3V6F0I 1NY477sfx0556KW
list #ModernTimes
send @Charlie #ModernTimes 5NCulMZYJbtmdm3rpqVrFxKGpKShHI
send @Jack #Infinity CEOaTILOEADbfRZLtxyamXH2LEnxoV
send @Charlie #ModernTimes aVcwKzl4bTem4sUzY55f08D6r488g0
send @Jack #Infinity 1IMIlih3w5R2n1si7DHknzykWfZQeI
send @Charlie #Infinity 9wrvhtyXQQLXlldvvz3YEy66Cvp uW
send @BuzzLightYear #Pumpkin uZ0tRwFZQOVU1sQUcREsVTy3bC5Zl5
send @Charlie #Infinity Tkci9wgbIuoIc534p37dazahwIkK83
send @BuzzLightYear #Infinity L2zderQeRiitWl0zQJvIwQgOC6MJi8
send @BuzzLightYear #ModernTimes K4ctP2A01ai6GzHRFFEghPDZZWd4at
send @BuzzLightYear #Pumpkin 9uMnSIxVpOx5JHupkGc4k1Sngu1OsQ
send @Charlie #Infinity YD1wTSGaI7dsmftvqkNfov4T7mRl3n
send @Charlie #Pumpkin AK kRu8fJWj27 MeU1hNxb3EqDrEzc
send @Charlie #ModernTimes WwjZrgoQAiQ15A0tyfsbEiHbVONN3n
send @BuzzLightYear #ModernTimes 0NC2zmNhvFEGDQO2wpeZjmTfT7Umxw
close @BuzzLightYear #ModernTimes
send @BuzzLightYear #Infinity jRXW6AGIqEh62CqMaKJiQ03WZIlQSp
send @BuzzLightYear #Infinity MTER07 Cs87ww14SY FbIIF8nKoGGE
close @BuzzLightYear #Pumpkin
send @BuzzLightYear #ModernTimes xHXLPcJsAUgdLgx4Tvji0ynxmNn0Wl
close @BuzzLightYear #Infinity
send @BuzzLightYear #ModernTimes nlLU9k9bFgXilAE80 FGuzrMgFv 6l
send @BuzzLightYear #ModernTimes q8ETGeTEQkZrEEKBVgDucyccgtWTKy
send @Jack #ModernTimes BwoxU3PPMjaU5uJQcOJZdc6wuBDiPL
close @BuzzLightYear #Pumpkin
send @BuzzLightYear #Infinity lg7VGOc0HVdwaLYCwXaPmNI7d5OEYG
send @Charlie #Pumpkin j5Wv5g2QGyato6LR19pBiykdbqIKcO
send @Charlie #Pumpkin 5ef5Oba6 u1iMeeDYIz7NTfBzGB2pJ
close @Charlie #Infinity
send @Charlie #Infinity HrdAkBn4cyhd TDJYoaGBJntKSTQFA
send @Charlie #ModernTimes gwE2qyneHbOx w1nvmxv3VYsfQ3xzY
send @Jack #Infinity 6jcgLIH6P3ZIsJw1nWLAD185RRgigW
send @Jack #ModernTimes  Q1K9Ilia4IY dPiNHd4L1tUXMk3Oc
list #Pumpkin
quit
