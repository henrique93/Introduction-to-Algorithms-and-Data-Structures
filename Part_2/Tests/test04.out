NULL
@Charlie #Infinity : uIJUgCA6HWtef8lYQL4RWViq8novxu
@Jack #Infinity : GUq76QjciIXwhBNujyI5WW9vczNvTj
NULL
@Charlie #ModernTimes : KuZW7NR UM6fRH0FM nRONkoZIM3Sz
@Charlie #Infinity : ycU3TXbHA8osUk35GdY6TXVhjFP3Lf
@Charlie #Infinity : wpOah3z1JPkdrqhJHDubCaODZ0KX9W
@BuzzLightYear #Infinity : IXpxzAlI9s47us5 KlQ7nPsSBKLbAp
@BuzzLightYear #Pumpkin : H5 fXwfMytU4S9eXQ7ymVNysPrTfLn
@Jack #Pumpkin : QGEt7glVESoDNLqeAHrrI6vgBYPhsg
@Charlie #Pumpkin : 7lipHexFcrjUxYBJ3xQnBPmL6pqX1y
@Charlie #Pumpkin : K06Vjxp MUAWcKYKVsYO65jj5egYiZ
@Jack #Pumpkin : dmJ0lk7yxmQhQ7P4o0EHf7QqfE6xVH
@Jack #ModernTimes : uBXVChPzDxV3RmoJlsmBAdzVcu31wS
@Jack #ModernTimes : uBXVChPzDxV3RmoJlsmBAdzVcu31wS
@Jack #ModernTimes : W6yhLxnF3V6F0I 1NY477sfx0556KW
@Charlie #Pumpkin : A4pBtdXJV2TMH7mza7elO9mQrKiLNE
@Charlie #Pumpkin : KcM752TmgcKgWPuwSlH0C7xm2elR r
@Jack #Pumpkin : wWZnGk 1sq4kX0UfBJPS3JnvpmeQ2a
@BuzzLightYear #Pumpkin : uZ0tRwFZQOVU1sQUcREsVTy3bC5Zl5
@BuzzLightYear #Pumpkin : 9uMnSIxVpOx5JHupkGc4k1Sngu1OsQ
@Charlie #Pumpkin : AK kRu8fJWj27 MeU1hNxb3EqDrEzc
@Charlie #Pumpkin : j5Wv5g2QGyato6LR19pBiykdbqIKcO
@Charlie #Pumpkin : 5ef5Oba6 u1iMeeDYIz7NTfBzGB2pJ
